hack/local-development/start-gardenlet

```diff
@@ -29,19 +31,69 @@ else
   trap cleanup_kubeconfig EXIT
 fi
 
+GARDEN_KUBECONFIG="${GARDEN_KUBECONFIG:-$kubeconfig}"
+
+# determine the seed to manage; if multiple seeds exist on the
+# garden, the SEED_NAME env variable must be set
+if [ -n "${SEED_NAME:-}" ]; then
+  seedConfig="$(kubectl --kubeconfig="$GARDEN_KUBECONFIG" get seed "$SEED_NAME" -o yaml)"
+else
+  seeds="$(kubectl --kubeconfig="$GARDEN_KUBECONFIG" get seeds -o yaml | yq eval '.items' -)"
+  case $(echo "$seeds" | yq eval 'length' -) in
+    0)
+      echo "No seed found in garden cluster."
+      exit 1
+      ;;
+    1)
+      seedConfig="$(echo "$seeds" | yq eval '.[0]' -)"
+      SEED_NAME="$(echo "$seedConfig" | yq eval '.metadata.name' -)"
+      ;;
+    *)
+      names="$(echo "$seeds" | yq eval --tojson --indent 0 '[.[].metadata.name]' -)"
+      echo "Multiple seeds exist in the garden, please specify \$SEED_NAME to select one of $names"
+      exit 1
+      ;;
+  esac
+fi
+
+# cleanup a bit to make the generated configuration file look a bit neater
+tmpFile="$DEV_DIR/seed-$SEED_NAME.yaml"
+echo "$seedConfig" | yq eval 'del(.apiVersion,.kind,.status,.metadata.annotations,.metadata.managedFields)' - > "$tmpFile"
```

Comment:
Can you also cleanup `metadata.{creationTimestamp,finalizers,generation,resourceVersion,uid}`?